packages/kuma-gui/features/mesh/dataplanes/Subscriptions.feature

```diff
@@ -44,6 +44,7 @@ Feature: dataplanes / subscriptions
     Then I click the "$about-dp-subscriptions a" element
     Then the URL contains "/meshes/default/data-planes/backend/overview/subscriptions"
     And the "$dp-subscriptions" element exists
+    And I wait for 500 ms
```

Comment:
I added one of these as I know all this subscriptions stuff is changing so I just add this in here for now, although I suspect there could be something weird going on with the fact that we are linking to a drawer from a drawer (in the same drawer position), something that we don't do anywhere else.